Source/RSP/Interpreter Ops.c

```diff
@@ -584,7 +584,7 @@ void RSP_Vector_VMUDN (void) {
 	for (el = 0; el < 8; el ++ ) {
 		del = EleSpec[RSPOpC.rs].B[el];
 
-		temp.UW = (DWORD)RSP_Vect[RSPOpC.rd].UHW[el] * (DWORD)(long)(RSP_Vect[RSPOpC.rt].HW[del]);
+		temp.UW = (uint32_t)((uint32_t)RSP_Vect[RSPOpC.rd].UHW[el]) * (uint32_t)(( int32_t)RSP_Vect[RSPOpC.rt].HW[del]);

```

Comment:
Now it is, yes, but before it wasn't.  It was for consistency back when I had it split to multiple lines, but since every multiply is in one single line now that type of consistency no longer looks as readable.